src/biokbase/narrative/tests/test_jobcomm.py

```diff
@@ -741,6 +743,198 @@ def test_get_job_logs_bad_job(self):
         )
         self.assertEqual("job_does_not_exist", msg["data"]["msg_type"])
 
+    # ------------------------
+    # Modify job update
+    # ------------------------
+    @mock.patch(
+        "biokbase.narrative.jobs.jobcomm.jobmanager.clients.get", get_mock_client
+    )
+    def test_modify_job_update__start__ok(self):
+        job_id_list = [JOB_COMPLETED, JOB_CREATED, BATCH_PARENT]
+        req = make_comm_msg("start_job_update", job_id_list, True)
+        self.jc._modify_job_updates(req)
+        msg = self.jc._comm.last_message
+        self.assertEqual(
+            {
+                "msg_type": "job_status",
+                "content": get_test_job_states(job_id_list)
+            },
+            msg["data"]
+        )
+        for job_id in ALL_JOBS:
+            if job_id in job_id_list:
+                self.assertEqual(
+                    self.jm._running_jobs[job_id]["refresh"],
+                    int(not saved_jobs[job_id]) + 1
+                )
+            else:
+                self.assertEqual(
+                    self.jm._running_jobs[job_id]["refresh"],
+                    int(not saved_jobs[job_id])
+                )
+        self.assertTrue(self.jc._lookup_timer)
+        self.assertTrue(self.jc._running_lookup_loop)
+
+    @mock.patch(
+        "biokbase.narrative.jobs.jobcomm.jobmanager.clients.get", get_mock_client
+    )
+    def test_modify_job_update__stop__ok(self):
+        job_id_list = [JOB_COMPLETED, JOB_CREATED, BATCH_PARENT]
+        req = make_comm_msg("stop_job_update", job_id_list, True)
+        self.jc._modify_job_updates(req)
+        for job_id in ALL_JOBS:
+            if job_id in job_id_list:
+                self.assertEqual(
+                    self.jm._running_jobs[job_id]["refresh"],
+                    max(int(not saved_jobs[job_id]) - 1, 0)
+                )
+            else:
+                self.assertEqual(
+                    self.jm._running_jobs[job_id]["refresh"],
+                    int(not saved_jobs[job_id])
+                )
+        self.assertIsNone(self.jc._lookup_timer)
+        self.assertFalse(self.jc._running_lookup_loop)
```

Comment:
Are these tests idempotent or will they need to be run in a specific order? It might be best to set the job update loop running at the beginning of each test so that there aren't any accidental effects if the tests get reordered at some point.

Reply:
I had it so everything was stopped in `setUp`. Adding `jc.start_job_status_loop` to the tests about stopping the loop.

Reply:
I actually have them stopped now in `setUp`